launch_testing/test/launch_testing/new_hooks/test_function_scope.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2021 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import launch
+import launch_testing
+
+import pytest
+
+
+@launch_testing.pytest.fixture()
+def ld():
```

Comment:
nitpick: `ld` isn't an ideal name, use `launch_description` if this is kind of a "callable object" or `get_launch_description` if a function name (use a verb for a function name)

Reply:
> nitpick: ld isn't an ideal name, use launch_description if this is kind of a "callable object" or get_launch_description if a function name (use a verb for a function name)

`launch_description` sounds fine, pytest.fixtures names are generally a noun phrase and not a verb, because that's also the name of the argument automatically passed to tests.

Reply:
fixed